### DISCUSSION TITLE
Minimal setup to get cli working with puppeteer

### DISCUSSION THREAD

**Discussion by raelb** [gildas-lormeau/SingleFile#989] [General]
Hi Gildas,
Can you help with some instructions on minimal setup to get cli working with puppeteer.
I've downloaded the SingleFile repo as a zip file. Extracted and run npm install

Then go to cli folder and type:
./single-file https://www.wikipedia.org wikipedia.html
in cmd line.

But that gives error:
Failed to launch the browser process! spawn chrome ENOENT

I installed puppeteer and puppeteer-core globally. Does the cli even need puppeteer to be installed? It looks like it is just using puppeteer-core. (I still get the above error).

Ideally, I do not want to install puppeteer or puppeteer-core globally. I just want to place it in local folder for the cli (i.e be self-contained) But not sure exactly where to place it. Ideally it would be working through installed chrome browser, so that it would pick up any cookies/password information automatically.

I'm working on Windows,
Thanks

**Comment by gildas-lormeau** [comment]
Hi @raelb, this error message means that puppeteer was not able to find the chrome executable. To fix this, you have to use the option `--browser-executable-path` and pass the full path to `chrome.exe`, e.g. `--browser-executable-path=C:\\Users\\Username\\AppData\\Local\\Google\\Chrome SxS\\Application\\chrome.exe`.

**Comment by raelb** [comment]
Thanks, that works. I notice that it will not use the cookies that are being used in the browser. Is there a way to achieve that?

**Comment by gildas-lormeau** [comment]
Sorry for the late answer, you can achieve this with `--browser-cookie` or `--browser-cookie-file`.
